lib/matplotlib/figure.py

```diff
@@ -1771,7 +1790,8 @@ def _normalize_grid_string(layout):
 
     def subplot_mosaic(self, mosaic, *, sharex=False, sharey=False,
                        width_ratios=None, height_ratios=None,
-                       empty_sentinel='.', subplot_kw=None, gridspec_kw=None):
+                       empty_sentinel='.', subplot_kw=None, gridspec_kw=None,
+                       per_subplot_kw=None):
```

Comment:
Since this is past `*`, it can be re-ordered to match the docstring?
```suggestion
                       empty_sentinel='.', subplot_kw=None, per_subplot_kw=None,
                       gridspec_kw=None):
```